src/core.js

```diff
@@ -138,8 +138,13 @@ RelativeFormat.prototype._compileMessage = function (units) {
     return new IntlMessageFormat(message, locales);
 };
 
-RelativeFormat.prototype._format = function (date) {
-    var now = dateNow();
+RelativeFormat.prototype._format = function (date, options) {
+    var now;
+    if (options && options.now) {
```

Comment:
I think that if `options.now` has a value other than `undefined`, then we should throw if that value is not finite, just like [we do for `date`](https://github.com/yahoo/intl-relativeformat/blob/master/src/core.js#L148-155).